CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Enclosure Group
 - FC Network
 - FCoE Network
+- Hypervisor Managers
```

Comment:
Please keep this resource under an unreleased version. Don't try to update the notes of a released version

Reply:
In the PR description, under issues, you have put a link to a PR. Is that PR related to this? Recheck it once.
And under issues, links to issues must be given but not a PR link.

Reply:
Done